### ISSUE TITLE
Smart rendering table

### ISSUE THREAD

**Issue by HLFH** [josdejong/jsoneditor#402]
Hi,

On your _Planned features_ document written in 2013, you have said that you wanted for JSONEditor to "improve performance for large json documents (smart rendering table?)."

Do you still plan to implement a smart rendering table?
Robomongo does it.

In fact, I have large JSON documents and a table view will greatly help me.

Thanks,

**Comment by josdejong** [comment]
Wow, I didn't remember the existence of that wiki page. I've removed it, it was terribly outdated.

Rendering large arrays doesn't have highest prio, but what I would love to implement is #297 for example.

**Comment by HLFH** [comment]
@josdejong :( Table mode within Robomongo is very useful. You don't intend to support it within your software?

**Comment by josdejong** [comment]
No, I was just cleaning up and thought that your idea is covered in #297. Just deduplicating.

**Comment by HLFH** [comment]
I don't think so. http://1.bp.blogspot.com/-D4-rom7-Cf8/VLhhmias15I/AAAAAAAAuII/k5o7nRTAuZ4/s1600/RM12.PNG
Let's say you manage a lot of financial transactions within your database and that you want to quickly compare several fields and values, I think in this case, a table view is well adapted. https://github.com/josdejong/jsoneditor/issues/297 is offtopic.

**Comment by josdejong** [comment]
Ah ok, sorry, misunderstood the issue. 

I thought you had this in mind: #297 but you have this in mind: #63. It's indeed a nice idea but for this project out of scope. That doesn't mean that anybody else can work on it of course :D

**Comment by josdejong** [comment]
Thinking about it, it's really cool indeed. Maybe we can pick this up in a later stadium. 

Currently I'm fully focusing on a complete rewrite (v6, see #337) though last months I wasn't able to work on it.

**Comment by HLFH** [comment]
In March 2017, [3T Software bought Robomongo](https://studio3t.com/press/3t-software-labs-acquires-robomongo-the-most-widely-used-mongodb-tool/). 

I would like to quote: 

> "We’re seeing tremendous growth in the MongoDB market, as more and more organisations invest in NoSQL database projects,” said Tomasz Naumowicz, co-founder of 3T Software Labs

Thank you for redirecting me to https://github.com/josdejong/jsoneditor/issues/337.
Some fintechs like Revolut use PostgreSQL, some others use MongoDB.
And I guess that in the next two/three years, some databases built around the GraphQL work will take the lead. I believe there is a real use case, even in the NoSQL scene, for a table mode with JSON.

I wish you a good night.

**Comment by josdejong** [comment]
ok ok you convinced me :) It's definitely a powerful view. It's just that with my limited amount of spare time I will not be able to work on it myself any time soon - want to focus on getting v6 out first. Any help would be welcome :).

Let's at least reopen this issue then and point to this one from #63.
